### ISSUE TITLE
Incompatible versions for `build-reqs` of starter

### ISSUE THREAD

**Issue by vivecalindahl-northvolt** [kedro-org/kedro#1345]
## Description
`kedro build-reqs` of starter "pandas-iris" fails due to unresolved requirements.

## Context
Cannot build requirements for new project, can't follow through with tutorials. 

## Steps to Reproduce
1. `kedro new --starter=pandas-iris`
2. Inside of new project, `kedro build-reqs`

## Expected Result
Requierments built.

## Actual Result
Error:

```
Could not find a version that matches jupyter_client<7.0,<8.0,>=4.1,>=5.1,>=5.3.4,>=6.1.1,>=7.0.0 (from -r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 7))
Tried: 4.0.0, 4.0.0, 4.0.0, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.1.1, 4.2.0, 4.2.0, 4.2.0, 4.2.1, 4.2.1, 4.2.1, 4.2.2, 4.2.2, 4.2.2, 4.3.0, 4.3.0, 4.3.0, 4.4.0, 4.4.0, 5.0.0, 5.0.0, 5.0.1, 5.0.1, 5.1.0, 5.1.0, 5.2.0, 5.2.0, 5.2.1, 5.2.1, 5.2.2, 5.2.2, 5.2.3, 5.2.3, 5.2.4, 5.2.4, 5.3.0, 5.3.0, 5.3.1, 5.3.1, 5.3.2, 5.3.2, 5.3.3, 5.3.3, 5.3.4, 5.3.4, 5.3.5, 5.3.5, 6.0.0, 6.0.0, 6.1.0, 6.1.0, 6.1.1, 6.1.1, 6.1.2, 6.1.2, 6.1.3, 6.1.3, 6.1.5, 6.1.5, 6.1.6, 6.1.6, 6.1.7, 6.1.7, 6.1.8, 6.1.8, 6.1.9, 6.1.9, 6.1.10, 6.1.10, 6.1.11, 6.1.11, 6.1.12, 6.1.12, 6.1.13, 6.1.13, 6.2.0, 6.2.0, 7.0.0, 7.0.0, 7.0.1, 7.0.1, 7.0.2, 7.0.2, 7.0.3, 7.0.3, 7.0.4, 7.0.4, 7.0.5, 7.0.5, 7.0.6, 7.0.6, 7.1.0, 7.1.0, 7.1.1, 7.1.1, 7.1.2, 7.1.2
Skipped pre-versions: 7.0.0a0, 7.0.0a0, 7.0.0a1, 7.0.0a1, 7.0.0rc0, 7.0.0rc0, 7.0.0rc1, 7.0.0rc1
There are incompatible versions in the resolved dependencies:
  jupyter_client<7.0,>=5.1 (from -r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 7))
  jupyter-client>=4.1 (from qtconsole==5.2.2->jupyter==1.0.0->-r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 6))
  jupyter-client<8.0 (from ipykernel==6.9.2->jupyter==1.0.0->-r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 6))
  jupyter-client>=7.0.0 (from jupyter-console==6.4.3->jupyter==1.0.0->-r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 6))
  jupyter-client>=5.3.4 (from notebook==6.4.9->jupyter==1.0.0->-r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 6))
  jupyter-client>=6.1.1 (from jupyter-server==1.15.2->jupyterlab==3.3.2->-r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 8))
  jupyter-client<7.0,>=5.1 (from kedro[pandas.csvdataset]==0.17.7->-r /Users/vivecalindahl/projects/kedro-tmp/new-kedro-project/src/requirements.in (line 9))
```

```
-- Separate them if you have more than one.
```

## Your Environment

* Kedro Version: 0.17.7
* Python 3.7.12 and ython 3.8.12

## Workaround
Added
`jupyter-console==6.4.0`
to requirements.in

**Comment by datajoely** [comment]
Hi @vivecalindahl-northvolt  you've got the right workaround - the fix for the next version has already been merged #1322

**Comment by datajoely** [comment]
For anyone else seeing this you can add `jupyter-console<6.4.3  # 6.4.3 requires jupyter_client>=7.0` in your `src/requirements.txt`

**Comment by lorenabalan** [comment]
Closing this as resolved in #1322
